include/net/mptcp.h

```diff
@@ -187,7 +187,7 @@ struct mptcp_tcp_sock {
 	u8	loc_id;
 	u8	rem_id;
 
-#define MPTCP_SCHED_SIZE 16
+#define MPTCP_SCHED_SIZE 128
```

Comment:
This is not really needed, right?

Reply:
Yes, you are right. This value is way to big. This change was introduced for other experimental scheduler implementations. I will revert this line.

Reply:
Well, 128 might be too big, but the ```blestsched_priv``` needs to be somehow optimized to become smaller.